src/CommunityToolkit.Maui/Views/AvatarView.shared.cs

```diff
@@ -0,0 +1,446 @@
+using System.ComponentModel;
+using CommunityToolkit.Maui.Core;
+using Microsoft.Maui.Controls.Internals;
+using Microsoft.Maui.Controls.Shapes;
+using Font = Microsoft.Maui.Font;
+
+namespace CommunityToolkit.Maui.Views;
+
+/// <summary>Avatar content view.</summary>
+public class AvatarView : Border, IAvatarView, IFontElement, ITextElement, IImageElement, ITextAlignmentElement, ILineHeightElement, ICornerElement
```

Comment:
I think all .NET MAUI interfaces should be parents for IAvatarView

Reply:
Some of them can't be, because they live in the `Maui.Controls` namespace, e.g. the [`IImageElement`](https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/IImageElement.cs)

Reply:
All the `Microsoft.Maui` interfaces were moved to the `IAvatarView`.  All the `Microsoft.Maui.Controls` interfaces can't be because of their namespace